pystiche_papers/ulyanov_et_al_2016/_loss.py

```diff
@@ -50,6 +50,29 @@ def content_loss(
     layer: str = "relu4_2",
     score_weight: Optional[float] = None,
 ) -> FeatureReconstructionOperator:
+    r"""Content_loss from :cite:`ULVL2016`.
+
+    Args:
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see
+            :ref:`here <table-hyperparameters-ulyanov_et_al_2016>` and below.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper. Additionally this
+            flag is used for switching between the github branches. For details see
+            :ref:`here <table-branches-ulyanov_et_al_2016>`.
```

Comment:
Basically we are switching between two reference implementations, correct? If that is the case we should write it like that since the fact that they live on two git branches is irrelevant.